config/default-lsst.psfex

```diff
@@ -11,8 +11,8 @@ PSF_ACCURACY    0.01            # Accuracy to expect from PSF "pixel" values
 #------------------------- Point source measurements -------------------------
  
 CENTER_KEYS     slot_Centroid_x,slot_Centroid_y # Catalogue parameters for source pre-centering
-PHOTFLUX_KEY    slot_ApFlux_flux    # Catalogue parameter for photometric norm.
-PHOTFLUXERR_KEY slot_ApFlux_fluxErr # Catalogue parameter for photometric error
+PHOTFLUX_KEY    base_CircularApertureFlux_9_0_flux      # Catalogue parameter for photometric norm
+PHOTFLUXERR_KEY base_CircularApertureFlux_9_0_fluxSigma # Catalogue parameter for photometric error
```

Comment:
I'm wondering if this is going to hurt our aperture corrections, and I'm thinking probably not because the PSFs will be tied to the 9-pixel aperture but we do the aperture correction later so everything ends up on the same system.  Right?